packages/localize/src/LocalizeManager.js

```diff
@@ -138,18 +163,41 @@ export class LocalizeManager extends LionSingleton {
     this.__storage[locale][namespace] = data;
   }
 
+  /**
+   * @param {RegExp | string} pattern
+   * @param {Function} loader
+   */
   setupNamespaceLoader(pattern, loader) {
     this.__namespacePatternsMap.set(pattern, loader);
   }
 
+  /**
+   * @param {Array<string | Object.<string, Function>>} namespaces
+   * @param {Object} [options]
+   * @param {string} [options.locale]
+   * @returns {Promise<Object>}
+   */
   loadNamespaces(namespaces, { locale } = {}) {
-    return Promise.all(namespaces.map(namespace => this.loadNamespace(namespace, { locale })));
+    return Promise.all(
+      namespaces.map(
+        /** @param {Object.<string, Function> | string} namespace */
+        namespace => this.loadNamespace(namespace, { locale }),
+      ),
+    );
   }
 
+  /**
+   * @param {Object.<string, Function> | string} namespaceObj
```

Comment:
this type get's used a lot... probably best to define it and then use it

either via a JsDoc TypeDef or within the `types.ts`... not sure what the best type would be here 😅

but it would be nice to just be able to write
```
* @param {NamespaceObj} namespaceObj
```

Reply:
Good point